backend/server/sql/migrations/20250426_drv.sql

```diff
@@ -0,0 +1,24 @@
+-- This is the minimal amount of information needed to identify a build
+-- This purposely tries to avoid details such as attr path which may
+-- differ (e.g. python3.pkgs.setuptools vs python3Packages.setuptools)
+CREATE TABLE IF NOT EXISTS Drv (
+    drv_path TEXT NOT NULL PRIMARY KEY,
+    system TEXT NOT NULL,
+    UNIQUE (drv_path) ON CONFLICT IGNORE
```

Comment:
```suggestion
    drv_path TEXT NOT NULL PRIMARY KEY ON CONFLICT IGNORE,
    system TEXT NOT NULL,
```

A bit unfortunate that SQlite allows nulls in primary keys by default :( Should probably check if I kept that in mind for the build model...